kratos/geometries/line_2d_2.h

```diff
@@ -792,6 +792,31 @@ class Line2D2 : public Geometry<TPointType>
     }
 
     /**
+     * It computes the normal of the geometry, if possible
+     * @return The normal of the geometry
+     */
+    virtual array_1d<double, 3> Normal() override
```

Comment:
It is defined in the base class, line 714, I added the calculus of the normal because to project you always need a direction of projection and the normal of the "plane". I defined the projection in such a way that the default direction of projection is a zero vector, so it takes the normal of the plane to project. I needed to add the calculus of the normal to reduce the number of arguments of the function.

In part this PR is created to continue the discussion that was stacked in #276 with practical solutions